Automattic/jetpack
# Admin Page: Move gulp-json-transform devDependency to dependencies in package.json

Fixes build for `production` environment when done in a brand new environment.

#### Changes proposed in this Pull Request:

Moves the `gulp-json-transform` from `devDependencies` to `dependencies`.

#### Testing instructions:

1. Run `npm run clean`
2. Run `npm run distclean`
3. Run `NODE_ENV=production npm run build`
4. Expect the build to work and find the build in `_inc/build`.
